src/backends/postgresql/session.cpp

```diff
@@ -29,6 +30,130 @@ void hard_exec(postgresql_session_backend & session_backend,
     postgresql_result(session_backend, PQexec(conn, query)).check_for_errors(errMsg);
 }
 
+// helper function to quote a string before sinding to PostgreSQL
+char * quote(PGconn * conn, const char *s, size_t len)
```

Comment:
Why deal with `char*` instead of just returning `std::string`? Using raw pointers is needlessly error-prone.

Reply:
I used the same approach (function declaration) as for MySQL. I will change it.